apps/cli/src/index.ts

```diff
@@ -49,4 +49,7 @@ program
   .option("-p, --platform <string>", "Selected platform")
   .action(returnLoggerMiddleware(checkToolsAsync));
 
+if (process.argv.length < 3) {
+  program.help()
+}
```

Comment:
AFAIK explicitly calling `.help()` is not necessary

Reply:
This is for the case when no arguments are passed in. Currently, it shows the help menu and then shows an error. This change gets rid of the error.

Reply:
ohh got it, thanks for the explanation